controller/api/destination/server.go

```diff
@@ -100,6 +102,12 @@ func (s *server) Get(dest *pb.GetDestination, stream pb.Destination_GetServer) e
 		log,
 	)
 
+	var token contextToken
+	b := dest.GetContextToken()
+	if err := json.Unmarshal([]byte(b), &token); err != nil {
+		log.Debugf("Error: %v", err)
+	}
+	log.Debugf("Context token: [ogToken=%s] [tokenStruct=%s] [ns=%s], [nodeName=%s]", dest.GetContextToken(), token, token.Ns, token.NodeName)
```

Comment:
logging the token all of these different ways is probably overkill.  I think we can either log just the raw token, or the parsed struct.

Reply:
Apologies for this! I left this in so that the logs I added in the PR description make a bit more sense. It was more to be let you (the reviewers) visualise the changes and manual tests a bit better. Should have been a bit more explicit, sorry! Will make sure we log a bit better in the actual implementation.